### PR TITLE
Add a backup mod for groups without active mods (HPC-7960)

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: lib/vsc/administration/ldapsync.py [hunk 1]
```diff
@@ -197,11 +197,15 @@ def sync_altered_groups(self, last, dry_run=True):
                 # if a 404 occured, the group is not an VO, so we skip this. Otherwise something else went wrong.
                 if err.code != 404:
                     raise
+            group_moderators = [str(m) for m in group.moderators]
```
Comment (stdweird) [inline lib/vsc/administration/ldapsync.py]: what's with all the `str`?

Comment (stdweird) [reply lib/vsc/administration/ldapsync.py]: nvm, it's just a bit odd (also `map(str, group.moderators)`, not usre it's that much more readable)

File: lib/vsc/administration/ldapsync.py [hunk 2]
```diff
@@ -197,11 +197,15 @@ def sync_altered_groups(self, last, dry_run=True):
...
+            institute_name = str(group.institute['name'])
+            if not group_moderators:
+                group_moderators = [VSC_CONFIG.backup_group_mods[str(group.institute['name'])]]
```
Comment (stdweird) [inline lib/vsc/administration/ldapsync.py]: the `str` here is probably not needed, so pls remove it

Comment (stdweird) [reply lib/vsc/administration/ldapsync.py]: and add a info statement that this is used.

File: lib/vsc/administration/ldapsync.py [hunk 3]
```diff
@@ -197,11 +197,15 @@ def sync_altered_groups(self, last, dry_run=True):
                 # if a 404 occured, the group is not an VO, so we skip this. Otherwise something else went wrong.
                 if err.code != 404:
                     raise
+            group_moderators = [str(m) for m in group.moderators]
+            institute_name = str(group.institute['name'])
+            if not group_moderators:
```
Comment (wpoely86) [inline lib/vsc/administration/ldapsync.py]: shouldn't this be done at the level of the accountpage?

Comment (stdweird) [reply lib/vsc/administration/ldapsync.py]: the group has no moderators, and that what's in the AP. we need a non-empty set of moderators for the ldap schema, that's why we will inject this here.
i prefer to keep the AP data clean and truthful

File: lib/vsc/administration/ldapsync.py (hunk 1 above)
Comment (itkovian) [reply lib/vsc/administration/ldapsync.py]: I thought we were not in favour of `map`, and rather have list comprehensions?

File: lib/vsc/administration/ldapsync.py (hunk 3 above)
Comment (itkovian) [reply lib/vsc/administration/ldapsync.py]: What @stdweird said.

File: lib/vsc/administration/ldapsync.py (hunk 1 above)
Comment (stdweird) [reply lib/vsc/administration/ldapsync.py]: ok, np

**Pull request merged**